app/javascript/components/Results/Results.tsx

```diff
@@ -128,15 +132,27 @@ export const Results = (props: ResultsProps) => {
                 </Grid>
               </GridContainer>
             );
-          })}
+          })) : (
+            <GridContainer className='result search-result-item'>
+              <Grid row>
+                <Grid tablet={{ col: true }}>
+                  <h4>Sorry, no results found for &#39;{props.query}&#39;. Try entering fewer or more general search terms.</h4>
+                </Grid>
+              </Grid>
+            </GridContainer>)}
         </div>
       </div>
-      {totalPages > 0 && 
+      {props.totalPages > 1 ? (
```

Comment:
This is where the logic gets complicated, all because of Bing. There's no obvious way to tell whether or not there's another page of results for Bing searches. The logic is:
```
if totalPages > 1, show pagination
else if totalPages = 0 or there's only one page of results, do not show pagination
else show pagination (this is a special case for Bing when you reach a page with no results)
```